presto-docs/src/main/sphinx/release/release-0.177.rst

```diff
@@ -0,0 +1,63 @@
+=============
+Release 0.177
+=============
+
+General
+-------
+
+* Fix correctness issue when performing range comparisons over columns of type ``CHAR``.
+* Fix correctness issue due to mishandling of nulls and non-deterministic expressions in
+  inequality joins unless ``fast_inequality_join`` is disabled.
+* Fix excessive GC overhead caused by lambda expressions. There are still known GC issues
+  with captured lambda expressions pending fix.
+* Prevent duplicate columns in ``CREATE TABLE``.
```

Comment:
I think we can remove this. It improves the error message by doing the check in the engine rather than leaving it up to the connector. Although if the connector doesn't check, the connector could have duplicate columns.